src/extensions/default/InlineBorderRadiusEditor/BorderRadiusEditor.js

```diff
@@ -0,0 +1,716 @@
+
+define(function(require, exports, module) {
+    "use strict";
+
+    var KeyEvent           = brackets.getModule("utils/KeyEvent"),
+        PreferencesManager = brackets.getModule("preferences/PreferencesManager"),
+        StringUtils        = brackets.getModule("utils/StringUtils"),
+        Strings            = brackets.getModule("strings"),
+        Mustache           = brackets.getModule("thirdparty/mustache/mustache"),
+        BorderRadiusUtils  = brackets.getModule("utils/BorderRadiusUtils");
+
+    /** Mustache template that forms the bare DOM structure of the UI */
+    var BorderRadiusTemplate = require("text!BorderRadiusEditorTemplate.html");
+    var DEFAULT_BORDER_RADIUS_VALUE = 0;
+    /**
+     * Box shadow editor control; may be used standalone or within an InlineBoxShadowEditor inline widget.
+     * @param {!jQuery} $parent  DOM node into which to append the root of the box-shadow editor UI
+     * @param {!{horizontalOffset: string, verticalOffset: string, blurRadius: string, spreadRadius: string, color: string}} values  Initial set of box-shadow values.
+     * @param {!function(string)} callback  Called whenever values change
+     */
+    function replaceAll(value,str1,str2,ignore){
```

Comment:
I don't feel like this is named well.  You have a much more complex regex in here than just replacing one string in another.  Can you add a comment to explain the regex, and what it's looking for, and also what this really does?